lib/test/cdo/test_config.rb

```diff
@@ -0,0 +1,56 @@
+require_relative '../test_helper'
+require 'cdo/config'
+require 'tempfile'
+
+class ConfigTest < Minitest::Test
```

Comment:
Some test cases I'd like to see:

* Resolving transitive dependencies successfully (like `test_circular_dependency` but without the circular-ness)
* `freeze` functionality

Reply:
- `test_erb` includes a transitive (`c -> b -> a`) dependency test along those lines
- I will add a test for `#freeze`